ambassador/ambassador/ir/ir.py

```diff
@@ -405,6 +405,7 @@ def features(self) -> Dict[str, Any]:
             od[key] = self.ambassador_module.get(key, False)
 
         od['xff_num_trusted_hops'] = self.ambassador_module.get('xff_num_trusted_hops', 0)
+        od['server_name'] = self.ambassador_module.get('server-name', 'envoy')
```

Comment:
Let's not actually return the custom server name in the features -- a boolean for whether or not it's been set explicitly should be fine (check out the `custom_ambassador_id` feature for another example).